### PR TITLE
Adding vcctl recipe

### PR DESCRIPTION
**Pull request by ThomasWollmann**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ThomasWollmann** [comment]
https://github.com/orgs/conda-forge/teams/staged-recipes This pull request is ready for a review.

**Comment by ThomasWollmann** [comment]
@conda-forge/staged-recipes This pull request is ready for a review.

**Comment by ThomasWollmann** [comment]
ping @bgruening

File: recipes/vcctl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  commands:
+    - vcctl
+
+about:
+  home: https://volcano.sh/en/docs/cli/
+  summary: A CLI for Volcano
+  description: |
+      A Command Line Interface (CLI) to manage resources of Volcano.
+  license: Apache-2.0
+  license_file: LICENSE
```
Comment (carterbox) [inline recipes/vcctl/meta.yaml:33]: Please do not check tasks on the TODO list you have not completed. This GO binaries has a bunch of statically linked dependencies. You need to collect the licenses of these other packages into a file and add this file to the license_file section.

There is a tool which can do collect the licenses for you automatically.

https://github.com/conda-forge/conda-forge.github.io/issues/1052

**Review by carterbox — CHANGES_REQUESTED** [review]

File: recipes/vcctl/meta.yaml (hunk 1 above)
Comment (ThomasWollmann) [reply recipes/vcctl/meta.yaml:33]: Thanks for pointing this out. Will do!

**Pull request closed without merging**